### PR TITLE
Feat/v1.13

### PR DESCRIPTION
**Pull request by clementnuss**

### CODE REVIEW

File: internal/kubenurse/server.go [hunk 1]
```diff
@@ -122,7 +122,8 @@ func New(ctx context.Context, c client.Client) (*Server, error) { //nolint:funle
 			bucket, e := strconv.ParseFloat(bucketStr, 64)
 
 			if e != nil {
-				log.Fatalf("couldn't parse one of the custom histogram buckets. error:\n%v", e)
```
Comment (zbindenren) [inline internal/kubenurse/server.go:125]: I would use `err` instead of `e`.

File: internal/kubenurse/server.go [hunk 2]
```diff
@@ -122,7 +122,8 @@ func New(ctx context.Context, c client.Client) (*Server, error) { //nolint:funle
...
+				slog.Error("couldn't parse one of the custom histogram buckets", "bucket", bucket, "err", e)
+				return nil, e
 			}
```
Comment (zbindenren) [inline internal/kubenurse/server.go:127]: should we exit here. if histogram is not parsable? earlier `log.Fatal`  was exiting here...

**Review by zbindenren — COMMENTED** [review]
Added some comments. Small stuff.

File: internal/kubenurse/server.go (hunk 1 above)
Comment (clementnuss) [reply internal/kubenurse/server.go:125]: right 👍🏼

File: internal/kubenurse/server.go (hunk 2 above)
Comment (clementnuss) [reply internal/kubenurse/server.go:127]: yeah, good point.

**Pull request merged**